.goreleaser.yaml

```diff
@@ -107,6 +107,34 @@ docker_signs:
     - '${artifact}'
     - "--yes" # needed on cosign 2.0.0+
 
+brews:
```

Comment:
Where does the name of this repo go? As in where is it stating that you have to run `brew install fga`?

Or is this tap a full thing, they have to install everything in it and have no choice to choose?